tensorflow/compiler/xla/service/hlo_graph_dumper.cc

```diff
@@ -1298,6 +1299,9 @@ NodeFilter MakeNodeRadiusAroundFilter(const HloInstruction* root,
     if (depth == radius) {
       continue;
     }
+    if (boundary.count(instr) != 0) {
```

Comment:
If we make `boundary` an absl::flat_hash_set<const HloInstruction*>, then you can use `absl::flat_hash_set::contains` instead of `count != 0`, which is kind of nice.  Up to you if you like it.